Source/Resources/Content/AadIam/MdmApplications/Microsoft Intune.json

```diff
@@ -8,6 +8,7 @@
     "termsOfUseUrl": "https://portal.manage.microsoft.com/TermsofUse.aspx"
   },
   "appId": "0000000a-0000-0000-c000-000000000000",
+  "description":"What is it: Turns on the ability manage a Windows device. \n Simeon Notes: This is a prerequisite for using Intune for managing devices.\n Learn More: https://docs.microsoft.com/en-us/mem/intune/apps/app-management, https://techcommunity.microsoft.com/t5/microsoft-intune/mdm-vs-mam/m-p/90906",
```

Comment:
**What does this do?**
Turns on the ability manage a Windows device, which is a prerequisite for using Intune for managing devices. 

**Why should you use this?**
If you want to use Intune to manage devices.

**What is the end user impact?**
Users will be able to manage devices (e.g. configure their machines) using Intune.  

**Learn more**
[MDM vs. MAM](https://techcommunity.microsoft.com/t5/microsoft-intune/mdm-vs-mam/m-p/90906)